code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -210,6 +219,20 @@
 	user.update_inv_head() //So the mob overlay updates
 	update_button_icons(user)
 
+/obj/item/clothing/head/helmet/space/plasmaman/proc/smash_headlamp()
+	if(!lamp_functional)
+		return
+	if(!helmet_on)
```

Comment:
can the helmet emit light if it's on the ground?

Reply:
Yes

Reply:
Then you should be able to lighteater it

Reply:
Oh. `helmet_on` means that the lamp itself is on. Nevermind. Badly named var lol